### DISCUSSION TITLE
Created an ios app with the ibm carbon design system

### DISCUSSION THREAD

**Discussion by brendanahart** [carbon-design-system/carbon#10742] [Show and tell]
Hi everyone, 

IBM Carbon has been huge for helping me develop an app that I created for searching, learning, and analyzing any trick that has been thrown in the action sports universe. Two weeks ago, I deployed to the app store! If anyone wants any insight into how I did it, feel free to comment on this discussion or reach out to me. Furthermore, here is the web app https://ride.infinitedegrees.xyz

![infd-athlete-competition-search-iphone11](https://user-images.githubusercontent.com/10844941/153777722-9145ed35-a2fb-4d02-83bd-98b70e50251f.png)
![infd-athlete-trick-search-iphone11](https://user-images.githubusercontent.com/10844941/153777724-04d71e46-3b6d-4b51-83d2-b8edea32b2d0.png)
![infd-basic-search-iphone11](https://user-images.githubusercontent.com/10844941/153777726-048e1ebc-f56c-4b7d-adee-a37a3aadf5b7.png)
![infd-ski-trick-detail-iphone11](https://user-images.githubusercontent.com/10844941/153777727-b0b3e860-9f35-49be-8474-74ab1188be4c.png)
![infd-trending-trick-iphone11](https://user-images.githubusercontent.com/10844941/153777728-b6f0e75b-846b-4001-8e12-2c4dab3e0cf9.png)

**Comment by tay1orjones** [comment]
Love this @brendanahart, thanks for sharing! It's fun to see atypical usages outside of the usual enterprise product software domain.  What led you to use Carbon?

**Reply by brendanahart** [reply]
A few things

- I am fluent in angular development and didn't want to learn react so I was looking for a design system that parlayed well with Angular. Wanted a design system because I've seen how much easier frontend development can be to create great looking websites.  
- Boostrap and Material Design are overused and look pretty bad imo. Tailwind UI costs money and isn't compatible with  angular
- IBM Carbon offers a robust grid system and combined with scss, you can organize components easily
- IBM carbon has some cool designed components! Especially for supporting pagination with tables. The icons are also fire! 
- IBM Carbon is open sourced and has great documentation. Great code examples too. Only thing that made it a bit difficult was that the story modules  here https://angular.carbondesignsystem.com/?path=/story/components-button--basic are encapsulated with angular components, but I just hit up the github to see how exactly the components were implemented

Originally, I had built out a web app with carbon, but then the product was needed for mobile. Carbon was able to adapt to that surprisingly well. 

Thanks team! Appreciate all of the development work you all do. Hopefully it helps propel Infinite Degrees to success!
